common/Log.cpp

```diff
@@ -407,6 +409,14 @@ namespace Log
 
         const std::string& getLevel() const { return _logLevel; }
 
+        void setUILogMergeInfo(bool mergeUiCmd, bool logTimeEndOfMergedUiCmd)
```

Comment:
Not entirely positive about having this functionality glupped into a set of methods with UI mangled in their name into the Logging class. Can we aggregate those as a sub-object of Log if we need to - and fetch that before calling methods that don't have UI in their name ? =) getUILog().setLogMergInfo(...) or whatever ?

Reply:
i made a new class inherited from the old:
static struct StaticUIHelper: StaticHelper
and there i renamed variables/methods to not have UI in it.
I hope this is what you wanted.